packages/http-core/test/integration/spec/request.js

```diff
@@ -9,9 +9,9 @@ import {HttpError, request} from '../..';
 import makeLocalUrl from '@ciscospark/test-helper-make-local-url';
 import sinon from '@ciscospark/test-helper-sinon';
 
-describe(`http-core`, function() {
-  this.timeout(30000);
-  describe(`request()`, () => {
+describe(`http-core`, () => {
+  describe(`request()`, function() {
+    this.timeout(30000);
```

Comment:
the `this` binding of arrow functions is the `this` at which they are defined, not the one within which they are invoked. in order to call `this.timeout` in a mocha spec, we need to use a function so that `this` gets bound.